arangod/Agency/State.cpp

```diff
@@ -1293,6 +1293,13 @@ index_t State::lastIndex() const {
   return _log.back().index;
 }
 
+// Index of first log entry
+index_t State::firstIndex() const {
+  MUTEX_LOCKER(mutexLocker, _logLock);
+  TRI_ASSERT(!_log.empty());
+  return _log.front().index;
```

Comment:
Should we not use `_cur` here instead, which is the same but readily available? Probably this does not matter.

Reply:
i would think that this improves readability. you would not have to know, what _cur is. compiler will have a significant acceleration for this i would expect